File: src/main/java/com/iflat/ss/bean/VrCode.java
package com.iflat.ss.bean;

/**
 * Created by tyriv on 2016/8/5.
 */
public class VrCode {

    private String id;
    private String riskLvl;
    private String code;
    private String description;
    private double amount;
    private double score;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getRiskLvl() {
        return riskLvl;
    }

    public void setRiskLvl(String riskLvl) {
        this.riskLvl = riskLvl;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public double getScore() {
        return score;
    }

    public void setScore(double score) {
        this.score = score;
    }
}

File: web/app/model/ss/VrCode.js
Ext.define('iFlat.model.ss.VrCode', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'vrCode.id', mapping: 'id', type: 'string'},
        {name: 'vrCode.riskLvl', mapping: 'riskLvl', type: 'string'},
        {name: 'vrCode.code', mapping: 'code', type: 'string'},
        {name: 'vrCode.description', mapping: 'description', type: 'string'},
        {name: 'vrCode.amount', mapping: 'amount', type: 'number'},
        {name: 'vrCode.score', mapping: 'score', type: 'number'},
    ]
});